src/utils/strings/__tests__/prose.js

```diff
@@ -0,0 +1,42 @@
+import expect from 'expect.js';
+
+import { formatListAsProse } from '../prose';
+
+describe('utils formatListAsProse()', () => {
+  it('converts non array arguments to string', () => {
+    expect(formatListAsProse(0)).to.eql('0');
+    expect(formatListAsProse(1)).to.eql('1');
+    expect(formatListAsProse({})).to.eql('[object Object]');
+    expect(formatListAsProse(() => {})).to.eql('() => {}');
+    expect(formatListAsProse((a, b) => b)).to.eql('(a, b) => b');
+    expect(formatListAsProse(/foo/)).to.eql('/foo/');
+    expect(formatListAsProse(null)).to.eql('null');
+    expect(formatListAsProse(undefined)).to.eql('undefined');
+    expect(formatListAsProse(false)).to.eql('false');
+    expect(formatListAsProse(true)).to.eql('true');
```

Comment:
^ all of this makes me sad 😢 What is the point of specifically string-ing stuff which isn't an array?